package.json

```diff
@@ -66,7 +66,8 @@
 		"unified": "9.1.0",
 		"unist-util-visit": "4.1.2",
 		"uvu": "0.5.2",
-		"vite": "5.4.11"
+		"vite": "5.4.11",
+		"glob": "11.0.0"
```

Comment:
Install this as a `devDependency` in `e2e/package.json` instead of the root `package.json`